### ISSUE TITLE
Elementor editor loading issue

### ISSUE THREAD

**Issue by mmhfarooque** [rupok/essential-addons-for-elementor-lite#274]
With the updated Elementor and Elementor pro ( 14th APril 2019) this plugin cause the Elementor editor infinite loading issue. Disabling this plugin solved the issue.

**Comment by jb205** [comment]
Same issue. It works with up to 11 elements activated but once I activate a 12th element, it gets stucks with spinning circle when loading.

**Comment by mmhfarooque** [comment]
> Same issue. It works with up to 11 elements activated but once I activate a 12th element, it gets stucks with spinning circle when loading.

did you counted the element from left to right? could you please name the element which activate the loading issue?

**Comment by jb205** [comment]
Here's what I did... New install w/ multisite that I've been working on getting off the ground.
Originally I had all elements enabled and it wouldn't load. So I deactivated ones I didn't think I'd use -- Still wouldn't load. Deactivated all -- loaded. Started activating ones I wanted a few at time until it stopped loading.

Then I deactivate one and activated another -- still loads. Tried a couple more times with same result.

So I can have up to 11 of the elements enabled in any order it seems. Once I try to enable 12, it stalls out on loading.
